poetry.lock

```diff
@@ -241,6 +257,25 @@ requests = ">=1.0.0"
 [package.extras]
 yaml = ["PyYAML (>=3.10,<5.3)"]
 
+[[package]]
+category = "main"
+description = "cryptography is a package which provides cryptographic recipes and primitives to Python developers."
+name = "cryptography"
```

Comment:
NB: `cryptography` is pulled in by `pytransferwise` in order to support [webhook signature verification](https://github.com/jayaddison/pytransferwise/blob/db86ad647f496687d3c6585d21b353b0c78305ff/pytransferwise/webhooks.py#L25).

This functionality isn't yet used in the website application but is important to include once we begin listening for incoming TransferWise webhook events.